src/verso-blog/VersoBlog/LiterateLeanPage.lean

```diff
@@ -394,6 +399,30 @@ partial def docFromMod (project : System.FilePath) (mod : String)
           }
         | other =>
           addBlock (← ofBlock helper other)
+    -- Lemma is purposefully unchecked, such that it can be matched on when a project
+    -- is dependent on mathlib
+    | ``declaration | `lemma =>
```

Comment:
This is making me think that this should be an attribute rather than a hardcoded list.

Reply:
Is there a way we can attach an attribute to the keyword lemma somehow (or can we avoid tagging every lemma in the book project)? If yes, I definitely agree.

If not, I think it depends on the aim of LiterateLeanPage: 

- If the goal is to best-effort render any existing lean file, then having to tag the custom-definitions you want to be converted is kind of rough. 
- If the goal is to provide an easy way to convert an existing lean file to something that can be rendered nicely, then I think introducing an attribute is fine.

I can totally get it if you don't want to have the hardcoded `lemma there in any case, so I'm also happy to remove it and maintain it as a patch right now for the analysis book project, since I'm currently doing that anyway.